internal/service/flexcluster/resource_schema.go

```diff
@@ -72,32 +89,53 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 						MarkdownDescription: "Public connection string that you can use to connect to this flex cluster. This connection string uses the `mongodb+srv://` protocol.",
 					},
 				},
-				Computed:            true,
+				Computed: true,
+				PlanModifiers: []planmodifier.Object{
+					objectplanmodifier.UseStateForUnknown(),
+				},
 				MarkdownDescription: "Collection of Uniform Resource Locators that point to the MongoDB database.",
 			},
 			"create_date": schema.StringAttribute{
-				Computed:            true,
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
 				MarkdownDescription: "Date and time when MongoDB Cloud created this instance. This parameter expresses its value in ISO 8601 format in UTC.",
 			},
 			"id": schema.StringAttribute{
-				Computed:            true,
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
 				MarkdownDescription: "Unique 24-hexadecimal digit string that identifies the instance.",
 			},
 			"mongo_db_version": schema.StringAttribute{
-				Computed:            true,
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
 				MarkdownDescription: "Version of MongoDB that the instance runs.",
 			},
 			"state_name": schema.StringAttribute{
-				Computed:            true,
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
```

Comment:
From our best practices doc I see we use this plan modifier for `computed attributes that will not have changes in their value`. I believe for state_name we could have different values.

Reply:
You are right! Will change it and also will confirm with the upstream team if there is any other attribute which might change after update

Reply:
Done